test/test.js

```diff
@@ -14,6 +14,19 @@ test('missing file', (assert) => {
     .on('error', (err) => {
       assert.ok(err, 'errored');
       assert.ok(err.message.indexOf('ENOENT') === 0, 'expected error');
+      assert.equals(err.code, 'EINVALID', 'expected error code');
```

Comment:
I'm not sure this one should count as an `EINVALID` error, because it actually means the file is missing, not that it's invalid. However, I'm guessing this bubbles up from `mapbox-file-sniff` which doesn't differentiate, so maybe there's nothing we can do?